docs/sources/operators-guide/configure-agent/language-support/go_push.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "Go (push mode)"
+menuTitle: "Go (push mode)"
+description: "Instrumenting Golang applications for continuous profiling"
+weight: 30
+---
+
+# Go (push mode)
+
+Pyroscope uses the standard `runtime/pprof` package to collect profiling data. Refer to
+[the official documentation](https://golang.org/doc/diagnostics#profiling) for details.
+
+## How to add Golang profiling to your application
+
+To start profiling a Go application, you need to include our go module in your app:
+```
+# make sure you also upgrade pyroscope server to version 0.3.1 or higher
+go get github.com/pyroscope-io/client/pyroscope
+```
+
+Note: If you'd prefer to use Pull mode you can find those docs here.
+
+Then add the following code to your application:
+
+```go
+package main
+
+import "github.com/pyroscope-io/client/pyroscope"
```

Comment:
@petethepig is this still fine after moving "github.com/pyroscope-io/client/pyroscope" to grafana org?